frontend/src/css/modules/base.scss

```diff
@@ -105,6 +105,10 @@ span.inline {
     position: absolute;
 }
 
+/*margin for column*/
+.ev-margin {
```

Comment:
@Ram81 Naah, that seems to be more general class name. Try to give some unique name. Think man ;)